### DISCUSSION TITLE
Rubrica Final del Proyecto Grupo 5. Auto

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Electronica-Microcontrolada/Grupo5#29] [Announcements]
Nota 1er Semana: 6,5 ( siete)   . 

Que tal chicos, trabajaron muy bien. 

a) Diagramaron el proyecto
b) hasta ahora son el unico grupo que explico la eleccion de su proyecto. 
c) Fueron juntando y organizando el bosquejo del trabajo
Pero aun no completaron los formularios que subio el profe Jorge Morales @JorEl057. En cuanto a lo de realizar el diagrama de gannt pueden utilizar un esquema grafico donde detallen el organigrama. ( lo pueden hacer en una hoja cuadriculada ). Por otro lado tampoco realizaron la presentacion audiovisual de su proyecto. Cuidado con esto, porque presentar correctamente un proyecto puede ser la diferencia entre conseguir los recursos (dinero) o no. 
Seguramente no van a tener mas inconvenientes en avanzar apropiadamente, saludos.

**Comment by Mauro-Martinez** [comment]
Hola Profe! Genial, Muchas Gracias en nombre del todo el Equipo!
Ya nos pondremos a corregir el diagrama de gannt y los de los formularios se nos paso por alto la verdad, procederemos a completar las fichas y subirlas.
Por ultimo, en el vídeo tenemos algunos inconvenientes debido a la inexperiencia en la edición pero estará listo en un par de días.
Saludos Cordiales!

**Comment by Gona79** [comment]
Nota de 3er Semana: 7(Siete)

Chicos el objetivo del desarrollo del proyecto era conocer las herramientas de:
Creacion de proyectos ( Gantt, AGILE, Proyectos)
Ideacion, desarrollo y prueba. Trabajaron muy al limite del tiempo, y no pareciera que fuese un trabajo con la carga bien distribuida. 
Pero principalmente el trabajo colaborativo.
Lo hicieron muy bien, adapten este trabajo en forma de plantilla** y les sera util para el resto de los proyectos de la cursada. Continuen asi.

** Deben crear los readme acordemente. Revisen los otros grupos. Y no olviden la creacion del proyecto Kanban.
